crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -190,11 +190,14 @@ pub enum TokenOwnerFindingStrategy {
 
 impl TokenOwnerFindingStrategy {
     /// Returns the default set of token owner finding strategies.
-    pub fn defaults_for_chain(chain_id: u64) -> &'static [Self] {
-        match chain_id {
-            1 => &[Self::Liquidity, Self::Blockscout, Self::Ethplorer],
-            100 => &[Self::Liquidity, Self::Blockscout],
-            _ => &[Self::Liquidity],
+    pub fn defaults_for_network(network: &network::Network) -> &'static [Self] {
+        match network {
+            network::Network::Mainnet => &[Self::Liquidity, Self::Blockscout, Self::Ethplorer],
```

Comment:
Why not `use network::Network` to make this less awkward? Given that this is not supposed to be a fundamental building block and the crate literally exports just a single type I think it's fine to use it without the absolute path.